src/types/onyx/Card.ts

```diff
@@ -92,5 +95,8 @@ type IssueNewCard = {
     currentStep: IssueNewCardStep;
 };
 
+/** List of Expensify cards */
+type ExpensifyCardsList = Record<string, Card>;
```

Comment:
I think this name should be a bit more generic still but we can change that later since its just a variable name